cumulus_library/actions/cleaner.py

```diff
@@ -76,7 +76,7 @@ def get_unprotected_stats_view_table(
 
 
 def clean_study(
-    manifest_parser: study_parser.StudyManifestParser,
+    manifest_parser: study_parser.StudyManifestParser | None,
```

Comment:
this allows the importer to not have to worry about setting up an empty parser.

Reply:
It does, but only if `prefix` is not supplied.

I missed updating the docstring here - i'll do that to clarify, and maybe and an error if both of these are null.

Reply:
Ah yeah doh - I missed its use :smile: